### PR TITLE
formatMetaV1 should be "inherited" by disk format structs

### PR DESCRIPTION
**Pull request by krishnasrinivas**
During the design discussion of format related code we had decided to include formatMetaV1 at the top of the backend format files. It was missed in formatXLV2 and formatXLV3

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |
| mint-gateway-azure.sh | :x: [more...](#86c7238d-756f-413b-a949-08c6f8cbbcc5) |
<a name="86c7238d-756f-413b-a949-08c6f8cbbcc5"></a>
### 6134-c01af2b/mint-gateway-azure.sh.log:
```
Running with
SERVER_ENDPOINT:      minikube:31598
ACCESS_KEY:           minioazure
SECRET_KEY:           ***REDACTED***
ENABLE_HTTPS:         0
SERVER_REGION:        us-east-1
MINT_DATA_DIR:        /mint/data
MINT_MODE:            full
ENABLE_VIRTUAL_STYLE: 0

To get logs, run 'docker cp 7898ceceb8da:/mint/log /tmp/mint-logs'
(1/13) Running awscli tests ... done in 2 minutes and 47 seconds
(2/13) Running aws-sdk-go tests ... done in 2 seconds
(3/13) Running aws-sdk-java tests ... done in 5 seconds
(4/13) Running aws-sdk-php tests ... done in 58 seconds
(5/13) Running aws-sdk-ruby tests ... done in 36 seconds
(6/13) Running mc tests ... done in 7 minutes and 40 seconds
(7/13) Running minio-dotnet tests ... done in 5 minutes and 48 seconds
(8/13) Running minio-go tests ... FAILED in 7 minutes and 41 seconds
{
  "args": {
    "policy": "{\"expiration\":\"2018-07-17T01:54:47.053Z\",\"conditions\":[[\"eq\",\"$bucket\",\"minio-go-test-oi22mhovpgcaphul\"],[\"eq\",\"$key\",\"vwnxrjt5gvarahqgu3mvxsv0wnsv2p\"],[\"eq\",\"$Content-Type\",\"binary/octet-stream\"],[\"eq\",\"$x-amz-meta-1tqnw9odg1qqbazew2c5vdgipbknw3\",\"h4cz456pmj15dwlzhfbjpyxdok035n\"],[\"content-length-range\", 10, 1048576]]}"
  },
  "duration": 371,
  "error": "400 Bad Request",
  "function": "PresignedPostPolicy(policy)",
  "message": "Http request failed",
  "name": "minio-go: testPresignedPostPolicy",
  "status": "FAIL"
}

Executed 7 out of 13 tests successfully.
```

File: cmd/format-xl.go [hunk 1]
```diff
@@ -107,9 +108,8 @@ type formatXLV2 struct {
 // In .minio.sys/multipart we have:
 // sha256(bucket/object)/uploadID/[xl.json, part.1, part.2 ....]
 type formatXLV3 struct {
-	Version string `json:"version"`
-	Format  string `json:"format"`
-	XL      struct {
+	formatMetaV1
```
Comment (balamurugana) [inline cmd/format-xl.go:111]: You would need to fix `format-xl_test.go` as well.

**Review by harshavardhana — APPROVED** [review]

**Review by balamurugana — APPROVED** [review]

**Comment by kannappanr** [comment]
ping @donatello

**Review by vadmeste — APPROVED** [review]
LGTM

**Pull request merged**